data/gamerules.txt

```diff
@@ -51,6 +51,11 @@ gamerules
 	# ALLOWABLE VALUES: any value between 0. and 1.
 	"universal frugal threshold" 0.75
 
+	# Controls whether disabled fighters get hit by stray projectiles not directly targeting them.
+	# DEFAULT: never
+	# ALLOWABLE VALUES: always, never, not player
```

Comment:
Seems like we're missing an "only player" option.

Reply:
So the player's fighters would get shot down, but NPC fighters wouldn't? I'm not sure how much use that would see.

Reply:
Ah. The meaning of the gamerule is a negation of how I would have described it. I would've described it as "all fighters dodge, no fighters dodge, or only the player's fighters dodge."

Checks out, I guess.

Reply:
Seeing as how the norm before this feature was for ships to get hit, I would actually prefer for this to be worded from the perspective of those fighters which don't get hit, instead of those fighters that do.

```
	# Controls which fighters, when disabled, will not be hit by stray projectiles that are not directly targeting them.
	# DEFAULT: all
	# ALLOWABLE VALUES: all, none, only player
```